public/css/main.css

```
@@ -0,0 +1,2092 @@
+    background-color: rgba(184, 26, 86, 0.1) !important;
+    border-bottom: 2px solid #B81A56 !important;
+    box-shadow: none !important;
+  }
+  
+  nav a.active::after {
+    display: none;
+  }
+  
+  nav a:focus, nav a.active:focus {
+    --tw-ring-opacity: 0 !important;
+    outline: none !important;
```

Stylelint (17.4.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)